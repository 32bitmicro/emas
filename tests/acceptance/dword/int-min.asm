.dword -2147483648
