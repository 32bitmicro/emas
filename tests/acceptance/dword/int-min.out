@ 0x0000 : 0x8000  /  100 000 0 000 000 000  /  32768
@ 0x0001 : 0x0000  /  000 000 0 000 000 000  /  0
